detection-rules/attachment_adobe_image_lure.yml

```diff
@@ -9,25 +9,26 @@ source: |
     or length(filter(attachments, .file_type != "pdf")) == 0
   )
   and all(body.links, .display_text is null)
-  and any(attachments,
-          any(ml.logo_detect(.).brands,
-              .name == "Adobe" and .confidence in ("medium", "high")
-          )
-          and any(file.explode(.),
-                  strings.ilike(.scan.ocr.raw,
-                                "*review*",
-                                "*sign*",
-                                "*view*",
-                                "*completed document*",
-                                "*open agreement*"
-                  )
-                  and (
-                    (length(body.links) > 0)
-                    or (length(.scan.url.urls) > 0 or length(.scan.pdf.urls) > 0)
-                  )
-          )
+  and (
+    any(file.explode(beta.message_screenshot()),
+        any(ml.logo_detect(beta.message_screenshot()).brands, .name == "Adobe")
+        and 0 < length(body.links) < 10
```

Comment:
this feels a little broad -- basically flagging on a message with an Adobe logo + a link? am i understanding correctly?

Reply:
and there's no display text on the links.
It seems broad, but it's fired a collective 180 times in testing, and there's only 4 hits with no other rule overlap. 

Of those 180 firings, 73 of them have actions confirming TP, (quarantine, trash, webhook) with 0 actions indicating FP's.